### ISSUE TITLE
GH-662 Custom Actions don't work as expected when parameters contain spaces

### ISSUE THREAD

**Issue by norio-nomura** [ForkIssues/Tracker#662]
Hi,

I have a Custom Action with:
```
$ defaults read com.DanPristupov.Fork customActions
(
        {
        name = "Open in Visual Studio Code - Insiders";
        parameters = "$path";
        path = "/usr/local/bin/code-insiders";
        showOutput = 1;
        target = repository;
    }
)
```
The "Open in Visual Studio Code - Insiders" fails when repository full path contains space.
Fork seems to splitting path by space.

Predefined action "Open in Visual Studio Code" does not have this issue.

I’m trying Catalina beta installing to additional APFS Volume in my Mac.
On that condition, the volume installed Mojave is mounted as `/Volumes/Macintosh HD` while using Catalina, a local repository path in Mojave volumes becomes like `/Volumes/Macintosh HD/Users/norio/github/…`.
This issue was found while using Fork in such repository.

**Comment by DanPristupov** [comment]
I've reproduced the problem and confirm that.
I'll try to fix this in the next update.

**Comment by norio-nomura** [comment]
Confirmed that this issue has been fixed on 1.0.81.
Thanks! 🙏
